docs/releases/network-upgrades.md

```diff
@@ -2,10 +2,70 @@
 
 For details on why and how network upgrades happen, along with explanations of the various types, please see the [Upgrades overview](#upgrades-overview) and [Upgrade methods](#upgrade-methods) sections further down.
 
+--8<-- "join-technical-mailing-list.md"
+
 ---
 
 ## Upcoming upgrades
 
+### Increased work difficulty
+
+To help ensure Quality of Service on the network by increasing the difficulty required for send and change blocks to be considered valid by the network (8x compared to current). To help offset the difficulty increase and add incentive to receive blocks so ledger pruning can be done more broadly in the future, the difficulty for receive blocks will simultaneously be reduced (1/8 compared to current).
+
+#### Transition details
+
+This upgrade is sometimes referenced as the epoch v2 upgrade and the relate events to complete are as follows:
+
+| Date | Type | Description |
+|------|------|-------------|
+| TBD  | Node release | Nano node V21.0 released which includes changes necessary for supporting new difficulty validation and generation |
+| TBD | <span class="no-break">v2 epoch blocks distribution start</span> | Distribution of v2 epoch blocks to all accounts to mark in the ledger the point at which the new work difficulty levels will be required. The start of this distribution process will occur once key services and over 90% of voting weight on the network has upraded. |
+| TBD | <span class="no-break">v2 epoch blocks distribution end</span> | Distribution of epoch blocks ends after all accounts are upgraded. |
+
+!!! warning "Nodes de-peered with epoch blocks"
+	Due to the nature of the work difficulty changes, any nodes not updated to V21.0 at the time of epoch block distribution will be de-peered from the network.
+
+#### Recommended preparations
+
+In order to best prepare for the transition to new thresholds, the following items should be considered:
+
+**Active difficulty**
+
+To programatically retrieve the current difficulty for any integrations doing work generation outside the node, the `network_minimum` field in [`active_difficulty`](/commands/rpc-protocol/#active_difficulty) RPC and [WebSocket topic](/integration-guides/websockets/#active-difficulty) will see a change from `ffffffc000000000` (pre-epoch v2 difficulty) to `fffffff800000000` (8x higher epoch v2 difficulty), an indication the epoch upgrade has begun.
+
+Once this occurs, send and change blocks should use this newly returned, higher threshold, and receive blocks can optionally use `fffffe0000000000` as the lower threshold going forward.
```

Comment:
The problem with passing this information now is that it's not valid as soon as the epoch ugprade begins, the account itself must be upgraded before it can use this lower threshold.

Reply:
Updated this overall section with nano-work-server and work generation guide callouts, but this detail on the lower receive threshold remains untouched.

Reply:
Moved to a collapsed callout with a note that this is not valid until epoch v2 distribution is done or accounts are upgraded.

Reply:
I noticed you included the difficulty increase (network_minimum) in here as well. That one however must be followed as soon as the distribution starts, right?